apps/desktop-wallet/src/api/queries/addressQueries.ts

```diff
@@ -33,21 +33,27 @@ export type AddressAlphBalancesQueryFnData = {
 export const addressAlphBalancesQuery = ({ addressHash, networkId, skip }: AddressLatestTransactionQueryProps) =>
   queryOptions({
     queryKey: ['address', addressHash, 'balance', 'ALPH', { networkId }],
```

Comment:
Nit: Here we pass `networkId` in an object, whereas in the PR you added it in one of the first keys in the array. We should maybe always do it the same way.

Reply:
Where did I add it as one of the first keys in the array?